ase/src/main/java/com/ase/ase/rest/controller/OverviewController.java

```diff
@@ -20,4 +22,18 @@ public ResponseEntity getOverview(){
         String o = casesTimelineRepository.getOverview();
             return ResponseEntity.ok(o);
     }
+
+    @CrossOrigin
+    @GetMapping(value = "/area-info/", produces = "application/json")
+    @ResponseBody
+    public ResponseEntity getRelativeAreaInfoBy(@RequestParam("area") Set<Integer> areas, @RequestParam("relative") boolean relative){
```

Comment:
Would be nice to have the relative parameter optional. By default, it should be false. You can use the field required=true in the @RequestParam. Check https://www.baeldung.com/spring-request-param for how to do it.